test/fgpu/test_pfb.py

```diff
@@ -82,5 +99,35 @@ def test_pfb_fir(
     fn()
     h_out = fn.buffer("out").get(command_queue)
     h_total_power = fn.buffer("total_power").get(command_queue)[()]
-    np.testing.assert_allclose(h_out, expected_out, rtol=1e-5, atol=1e-6 * 2**dig_sample_bits)
+    np.testing.assert_allclose(h_out, expected_out, rtol=1e-5, atol=1e-6 * 2**input_sample_bits)
     assert h_total_power == expected_total_power
+
+
+@pytest.mark.parametrize("unzip_factor", [1, 2, 4])
+def test_pfb_fir_complex(context: AbstractContext, command_queue: AbstractCommandQueue, unzip_factor: int) -> None:
+    taps = 16
+    spectra = 3123
+    channels = 4096
```

Comment:
Once again, not at all priority, but should we earmark a general 'consolidation of variable declarations'? Is it worth declaring these as fixtures?